doc/radosgw/admin.rst

```diff
@@ -311,7 +311,7 @@ To add administrative capabilities to a user, execute the following::
 You can add read, write or all capabilities to users, buckets, metadata and 
 usage (utilization). For example::
 
-	--caps="[users|buckets|metadata|usage|zone|amz-cache|info|bilog|mdlog|datalog|user-policy|oidc-provider|roles|ratelimit]=[*|read|write|read, write]"
+	--caps="[users|keys|buckets|metadata|usage|zone|amz-cache|info|bilog|mdlog|datalog|user-policy|oidc-provider|roles|ratelimit]=[*|read|write|read, write]"
```

Comment:
@cbodley do you think it would be worth documenting what each of these caps does at a high level in the docs? There's nothing like that so far, though in the admin api does reference which caps are necessary for certain operations.

Reply:
if the adminapi docs are consistent about showing the required caps, i think that's probably all the documentation we need